crates/types/src/motion_file.rs

```diff
@@ -50,68 +51,55 @@ where
     Ok(Duration::from_secs_f32(f32::deserialize(deserializer)?))
 }
 
+impl_Interpolate!(f32, Joints, std::f32::consts::PI);
```

Comment:
Replace qualified path with use?

```suggestion
impl_Interpolate!(f32, Joints, PI);
```

Reply:
Done